examples/hyper-1-0/Cargo.toml

```diff
@@ -6,9 +6,10 @@ publish = false
 
 [dependencies]
 axum = { path = "../../axum" }
-hyper = { version = "1.0.0-rc.3", features = ["full"] }
+hyper = { version = "=1.0.0-rc.4", features = ["full"] }
+hyper-util = { git = "https://github.com/hyperium/hyper-util", features = ["full"] }
```

Comment:
I'd recommend pinning git dependencies to a specific rev, especially if there's no git-tracked lockfile.